examples/text_labels_and_annotations/text_alignment.py

```diff
@@ -47,24 +49,24 @@
         verticalalignment='top',
         transform=ax.transAxes)
 
-ax.text(left, 0.5*(bottom + top), 'right center',
+ax.text(left, 0.5 * (bottom + top), 'right center',
```

Comment:
In the above line the operator with the lowest priority is the plus operator around which whitespace is added. PEP8 does not say anything about adding whitespace to all operators independent of their priorities.